### PR TITLE
[Kotlin Migration] ThreadUtils

### PR DESCRIPTION
**Pull request by codingtosh**
[Kotlin Migration] ThreadUtils
Package: com.ichi2.anki.testutil

### CODE REVIEW

**Review by david-allison — APPROVED** [review]

File: AnkiDroid/src/androidTest/java/com/ichi2/anki/testutil/ThreadUtils.kt [hunk 1]
```diff
@@ -13,26 +13,26 @@
...
-public class ThreadUtils {
-    public static void sleep(int timeMs) {
+object ThreadUtils {
+    @JvmStatic
+    fun sleep(timeMs: Int) {
         try {
-            Thread.sleep(timeMs);
-        } catch (InterruptedException e) {
-            throw new RuntimeException(e);
+            Thread.sleep(timeMs.toLong())
```
Comment (mikehardy) [inline AnkiDroid/src/androidTest/java/com/ichi2/anki/testutil/ThreadUtils.kt:22]: this toLong and the one on L33 should be removed I think?

**Review by mikehardy — CHANGES_REQUESTED** [review]
the toLong's then good to go

File: AnkiDroid/src/androidTest/java/com/ichi2/anki/testutil/ThreadUtils.kt (hunk 1 above)
Comment (codingtosh) [reply AnkiDroid/src/androidTest/java/com/ichi2/anki/testutil/ThreadUtils.kt:22]: The doc specifies the params as long, results in a type mismatch error if toLong() is removed. 
https://developer.android.com/reference/java/lang/Thread#join(long)
https://developer.android.com/reference/java/lang/Thread#sleep(long)
PTAL

Comment (mikehardy) [reply AnkiDroid/src/androidTest/java/com/ichi2/anki/testutil/ThreadUtils.kt:22]: If they have to exist, they have to exist :-)

**Review by mikehardy — APPROVED** [review]

**Pull request merged**